packages/lib/src/formatters/configFormatters.ts

```diff
@@ -78,8 +79,7 @@ const jsonToHelmValues: FormatterFunction = ({ json }) => {
 };
 
 const configFormatters: Record<ConfigFormat, FormatterFunction> = {
-  JSON: ({ json }) => JSON.stringify(json, null, 2),
-  CompactJSON: ({ json }) => JSON.stringify(json),
+  JSON: ({ json }) => JSON.stringify(json),
```

Comment:
@jamesL92 I would destruct the `pretty` option here, as the formatter should handle it accordingly. If a formatter doesn’t have a "pretty" mode, I think the most intuitive approach would be to simply ignore the value and proceed with the standard formatting.

Reply:
Good call - this would be a better approach as we expand to more flags.